src/coprocessor/dag/expr/builtin_miscellaneous.rs

```diff
@@ -118,6 +151,60 @@ mod test {
         }
     }
 
+    #[test]
+    fn test_inet_aton() {
+        let cases = vec![
+            (Datum::Bytes(b"0.0.0.0".to_vec()), Datum::I64(0)),
+            (
+                Datum::Bytes(b"255.255.255.255".to_vec()),
+                Datum::I64(4294967295),
+            ),
+            (Datum::Bytes(b"127.0.0.1".to_vec()), Datum::I64(2130706433)),
+            (
+                Datum::Bytes(b"113.14.22.3".to_vec()),
+                Datum::I64(1896748547),
+            ),
+            (Datum::Bytes(b"".to_vec()), Datum::Null),
+            (Datum::Bytes(b"0.0.0.256".to_vec()), Datum::Null),
+            (Datum::Bytes(b"127.0.0,1".to_vec()), Datum::Null),
+            (Datum::Null, Datum::Null),
```

Comment:
Please add tests for "1", "0.1.2", "0.1.2.3.4", "0.1.2..3", ".0.1.2.3", "0.1.2.3.", "1.-2.3.4". Notice that:

```
mysql> select Inet_Aton('0.1.2');
+--------------------+
| Inet_Aton('0.1.2') |
+--------------------+
|              65538 |
+--------------------+
1 row in set (0.00 sec)

mysql> select Inet_Aton('1');
+----------------+
| Inet_Aton('1') |
+----------------+
|              1 |
+----------------+
1 row in set (0.00 sec)
```